features/fixtures/mazerunner/app/src/main/AndroidManifest.xml

```diff
@@ -2,9 +2,12 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.bugsnag.android.mazerunner">
 
+    <!-- always enable GWP ASAN tool to detect native memory bugs. see
```

Comment:
I'd suggest adding a similar comment above the Android 11 steps in the pipeline, to make it stand out that we're using gwp-asan and that failures could occur as a result, but that the device logs need to be inspected to determine why.

Reply:
Added a comment to the Android 11 steps in the pipeline yml.